Solutions/Sophos XG Firewall/Package/createUiDefinition.json

```diff
@@ -64,35 +95,18 @@
             "name": "workbooks-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This Azure Sentinel Solution installs workbooks. Workbooks provide a flexible canvas for data monitoring, analysis, and the creation of rich visual reports within the Azure portal. They allow you to tap into one or many data sources from Azure Sentinel and combine them into unified interactive experiences.",
+              "text": "This Microsoft Sentinel Solution installs workbooks. Workbooks provide a flexible canvas for data monitoring, analysis, and the creation of rich visual reports within the Azure portal. They allow you to tap into one or many data sources from Microsoft Sentinel and combine them into unified interactive experiences."
```

Comment:
Please verify if workbook content is correct?

In text change document I see below

![image](https://user-images.githubusercontent.com/107389644/175524770-07f81bea-1ab4-423c-8190-d9cbecddb3f0.png)